Makefile

```diff
@@ -93,7 +93,8 @@ manager: generate fmt vet
 run: export WATCH_NAMESPACE=$(LOCAL_RUN_NAMESPACE)
 run: export THREESCALE_DEBUG=1
 run: generate fmt vet manifests
-	$(GO) run ./main.go --zap-devel
+	@-oc process THREESCALE_VERSION=$(VERSION) -f config/requirements/operator-requirements.yaml | oc apply -f - -n $(WATCH_NAMESPACE)
```

Comment:
just wondering how to run the debug as this command won't be included . 

guess we can set `"PREFLIGHT_CHECKS_BYPASS", "true"` in the debug configuration beside `WATCH_NAMESPACE`

WDYT

Reply:
@austincunningham if you want debug, you can either disable preflights like you mentioned above, or, install via OLM and scale down the operator deployment.